tests/components/shelly/test_climate.py

```diff
@@ -533,7 +556,7 @@ async def test_block_restored_climate_auth_error(
     await hass.config_entries.async_setup(entry.entry_id)
     await hass.async_block_till_done()
 
-    assert entry.state is ConfigEntryState.LOADED
+    assert entry.state == ConfigEntryState.LOADED
```

Comment:
When comparing an enum it is better to use `is`:
```suggestion
    assert entry.state is ConfigEntryState.LOADED
```
I would actually fix this in other places instead of changing this to `==`